testdata/Makefile

```diff
@@ -29,3 +41,42 @@ loader-%-eb.elf: loader.c
 
 %-eb.elf : %.c
 	$(CLANG) $(CFLAGS) -mbig-endian -c $< -o $@
+
+help: ## Display this help.
+	@grep -E '^[a-zA-Z_-]+:.*## .*$$' $(MAKEFILE_LIST) | \
+		awk 'BEGIN {FS = ":.*## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+else
+
+# If the first make goal contains `docker`, only declare Docker-related targets.
+
+.PHONY: docker docker-shell
+
+# Get the absolute path and uid/gid of the repository on disk.
+GITDIR := $(shell git rev-parse --show-toplevel)
+UIDGID := $(shell stat -c '%u:%g' ${GITDIR})
+
+# Pop off the 'docker' goal so the remaining goals can be passed to
+# the Dockerized make.
+PARAMS := $(filter-out docker,$(MAKECMDGOALS))
+
+DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+REPO := $(shell cat ${DIR}/docker/REPO)
+VERSION := $(shell cat ${DIR}/docker/VERSION)
+
+docker: ## Wrap the remaining arguments into a Dockerized 'make'.
+	docker run --rm --user "${UIDGID}" \
+		-v "${GITDIR}":/ebpf -w /ebpf/testdata \
```

Comment:
Use `$(CURDIR)` again, something like:
```
-v "${CURDIR}":/work -w /work
```

That way the invocation

Reply:
Let's keep the conversation on `${CURDIR}` centered around a single thread